### PR TITLE
Master to Live

### PR DESCRIPTION
**Pull request by chrisda**

### CODE REVIEW

**Comment by chrisda** [comment]
@v-dihans, @traya1, that Get-CSTeamsCallHold Policy is not mine, and I did not touch it. Nevertheless, I went in and attempted to fix its formatting issues.

**Comment by traya1** [comment]
Thanks Chris,

I've left a comment in the source PR about these items. I'll take care of the warnings shortly.

File: skype/skype-ps/skype/Get-CsTeamsCallHoldPolicy.md [hunk 1]
```diff
@@ -32,7 +32,8 @@ Get-CsTeamsCallHoldPolicy [-Tenant <System.Guid>] [-Filter <String>] [-LocalStor
 ## DESCRIPTION
 Teams call hold policies are used to customize the call hold experience for teams clients.
 When Microsoft Teams users participate in calls, they have the ability to hold a call and have the other entity in the call listen to an audio file during the duration of the hold.
-Assinging a teams call hold policy to a user sets an audio file to be played during the duration of the hold. 
+Assigning a Teams call hold policy to a user sets an audio file to be played during the duration of the hold. 
+
```
Comment (ezatullah33) [inline skype/skype-ps/skype/Get-CsTeamsCallHoldPolicy.md:38]: my life is danger
I hope you will helpe mi with this
As long as I can live a comfortable
I work for an organization run by the 
united states
I have four children and a disabled
brother making the whele family 7 people
please look at my reqest and email give me
the necessary information thank you

Comment (ezatullah33) [reply skype/skype-ps/skype/Get-CsTeamsCallHoldPolicy.md:38]: yellopoint0017@gmail.com

**Pull request merged**